recipes/ipyparallel/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "5.0.1" %}
+
+package:
+  name: ipyparallel
+  version: {{ version }}
+
+source:
+  fn: ipyparallel-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/ipyparallel/ipyparallel-{{ version }}.tar.gz
```

Comment:
So, I'm not sure that I like that as the default. Here is my reasoning (partially motivated by @ocefpaf).

When one makes a PyPI release, it is basically impossible to overwrite the file submitted with the release. However, with GitHub tags this is trivial. Now it's true that people shouldn't do that, but the fact that it is possible and one might be tempted to is enough of a reason for me to steer clear.

The checksum sort of prevents against this, but then if it does become inaccurate we aren't sure if the release has been modified in some way, if we are having connectivity issues, or worse someone is tampering with what we are receiving. If we determine it has been modified (probably by repeated attempts), it is hard for us to tell as the old tag doesn't exist any more and we might not know the commit (assuming it even still exists).

TL;DR PyPI gives us some nice guarantees that GitHub doesn't. These guarantees protect us and our users. We should really embrace them.